packages/shared/src/modules/groups/libs/enums/group-validation-message.enum.ts

```diff
@@ -0,0 +1,10 @@
+import { GroupValidationRule } from "./group-validation-rule.enum.js";
+
+const GroupValidationMessage = {
+	NAME_REQUIRED: "Project name is required.",
+	NAME_TOO_LONG: `Name is too long (> ${String(GroupValidationRule.NAME_MAXIMUM_LENGTH)} symbols).`,
+	PERMISSION_IDS_REQUIRED: "At least one permission ID is required.",
```

Comment:
We should allow to create groups without permissions

Reply:
![Screenshot from 2024-08-27 13-02-30](https://github.com/user-attachments/assets/9a69bed3-9c09-459c-9edd-150ca1afc0f1)